packages/document/main-doc/docs/en/guides/advanced-features/ssr.mdx

```diff
@@ -39,24 +40,25 @@ export default () => {
 };
 ```
 
-Modern.js break the traditional SSR development model and provide users with a user-friendly SSR development experience.
+Modern.js breaks the traditional model of server-side rendering (SSR) development and offers users a more user-friendly SSR development experience.
 
-And it provides elegant degradation processing. Once the SSR request fails, it will automatically downgrade and restart the request on the browser side.
+This feature offers elegant degradation processing. In case the SSR request fails, it will automatically downgrade and restart the request on the browser side.
```

Comment:
In case : describe a precautionary measure taken in anticipation of a future event. 
eg. In case it rains, I'll bring an umbrella.

Once: refers to a specific point in time or an action that will happen after a certain condition is met.
eg. Once you finish your homework, you can watch TV.

But, I think 'if' is better here...